README.md

```diff
@@ -8,33 +8,36 @@
 
 Matterpoll is a plugin for [Mattermost](https://mattermost.com/). It allows users to create poll by using a slash command.
 
+![Matterpoll plugin screenshot](images/screenshot.png)
 
 ## Installation
 
 1. Go to the [releases page of this GitHub repository](https://github.com/matterpoll/matterpoll/releases/latest) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. You should set **Enable integrations to override usernames** and **Enable integrations to override profile picture icons** in **System Console > Custom Integrations** to `true`.
-
+2. In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.
```

Comment:
Sorry, my comment was ambiguous. I was suggesting to replace the whole "Installation" section with `In Mattermost, open **Main Menu > Plugin Marketplace** to install and configure Matterpoll.`

Reply:
I missed that..🤦  Thanks!